### PR TITLE
Fix issues reported from Windows Server v23H2 testing

### PR DESCRIPTION
**Pull request by Tomorrow9**

### CODE REVIEW

**Review by 123lzxm — DISMISSED** [review]
LGTM, thanks.

**Review by keirazhang — DISMISSED** [review]
LGTM

File: linux/network_device_ops/check_pvrdma_support_status.yml [hunk 1]
```diff
@@ -3,6 +3,17 @@
 ---
 # Check whether target VM supports PVRDMA testing
 #
+- name: "Skip test case"
+  include_tasks: ../../common/skip_test_case.yml
+  vars:
+    skip_msg: >-
+      Test case '{{ ansible_play_name }}' is skipped because vCenter server is not configured,
+      while VM clone requires to connect to it in this test case.
```
Comment (keirazhang) [inline linux/network_device_ops/check_pvrdma_support_status.yml:11]: Please change 
```
while VM clone requires to connect to it in this test case.
``` 
to 
```
which is required for setting up vSphere Distributed Switch networking and cloning PVRDMA client VM.
```

Comment (Tomorrow9) [reply linux/network_device_ops/check_pvrdma_support_status.yml]: Sure, thanks.

**Review by 123lzxm — APPROVED** [review]
LGTM, thanks.

**Pull request merged**